test/cljs/unit/ataru/dob_test.cljs

```diff
@@ -0,0 +1,17 @@
+(ns ataru.dob-test
+  (:require [ataru.dob :as dob]
+            [cljs.test :refer-macros [deftest is testing]]))
+
+(deftest dob-identification
+  (doall
+    (for [[dob expected] [["1.1.2010" true]
+                          ["01.01.2010" true]
+                          ["1.01.2010" true]
+                          ["01.1.2010" true]
+                          ["1.13.2010" true] ; purposefully true
```

Comment:
Improved the comment there. Problem is that `cljs.time` library automatically converts `1.13.2010` to `1.1.2011` when parsing the date. This means that invalid dates like `1.13.2010` or `40.10.2010` would have to be parsed manually, outside cljs.time.

At least for I thought this is a bit too much effort to implement properly.

Please note that on the clj side, in `ataru.dob-spec`, the same namespace is tested with different expectations.